package.json

```diff
@@ -27,5 +27,8 @@
   },
   "devDependencies": {
     "lerna": "^4.0.0"
+  },
+  "dependencies": {
```

Comment:
This should be installed in the common package's package.json file instead of the root.